### PR TITLE
Making CI great again

### PR DESCRIPTION
**Pull request by ajsantander**

### CODE REVIEW

File: packages/synthetix-main/scripts/deploy.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const {
  */
 module.exports.deploy = async function deploy(runtime, prefix, modules) {
   if (runtime?.provider) {
+    console.log('overriding provider');
```
Comment (ajsantander) [inline packages/synthetix-main/scripts/deploy.js:13]: A couple of console logs left over

File: packages/synthetix-main/test/integration/bootstrap.ts [hunk 2]
```diff
@@ -33,47 +34,51 @@ export function bootstrap() {
...
+        _provider
+      ),
+      new hre.ethers.Wallet(
+        '0x47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a',
+        _provider
+      ),
+      new hre.ethers.Wallet(
+        '0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba',
+        _provider
+      ),
```
Comment (ajsantander) [inline packages/synthetix-main/test/integration/bootstrap.ts:60]: Why do we still need to hardcode signers in our tests?

**Review by ajsantander — COMMENTED** [review]
Looks good, left a couple of comments

File: packages/synthetix-main/test/integration/bootstrap.ts (hunk 2 above)
Comment (mjlescano) [reply packages/synthetix-main/test/integration/bootstrap.ts:60]: Yes, as we are using them for testing.

**Pull request merged**